examples/istio-with-separate-cert/zarf.yaml

```diff
@@ -0,0 +1,67 @@
+kind: ZarfPackageConfig
```

Comment:
I do really like that this example shows as little as possible to highlight the istio/secret stuff.